### PR TITLE
First pass at the single stack experience

### PR DESCRIPTION
**Pull request by MartinJibber**
https://user-images.githubusercontent.com/88675954/158274484-e4961141-9c1b-403f-884c-621bbc088b1a.mp4

### CODE REVIEW

File: Shared/Flows/Onboarding/Welcome/WelcomeViewController.swift [hunk 1]
```diff
@@ -87,7 +87,7 @@ class WelcomeViewController: DiffableCollectionViewController<MessageSequenceSec
     }
     
     override func getAnimationCycle(with snapshot: NSDiffableDataSourceSnapshot<MessageSequenceSection, MessageSequenceItem>) -> AnimationCycle? {
-        let count = (snapshot.numberOfItems(inSection: .topMessages) + snapshot.numberOfItems(inSection: .bottomMessages)) - 1
+        let count = snapshot.numberOfItems(inSection: .messages) - 1
```
Comment (MartinJibber) [inline Shared/Flows/Onboarding/Welcome/WelcomeViewController.swift]: There's only one type of message section now.

File: Shared/Flows/Onboarding/Welcome/WelcomeViewController.swift [hunk 2]
```diff
@@ -174,26 +174,19 @@ class WelcomeViewController: DiffableCollectionViewController<MessageSequenceSec
...
-                                                                     showDetail: false))
-                } else {
-                    otherMessages.append(MessageSequenceItem.message(cid: cid, messageID:
-                                                                        message.id,
-                                                                     showDetail: false))
-                }
+                guard message.authorId == PFConfig.current().adminUserId else { return }
+                benjiMessages.append(MessageSequenceItem.message(cid: cid,
+                                                                 messageID: message.id,
+                                                                 showDetail: false))
```
Comment (MartinJibber) [inline Shared/Flows/Onboarding/Welcome/WelcomeViewController.swift]: If there's only one stack, I assume we'll just show the Benji message for now?

File: Shared/UI/Views/MessageContentView/MessageContentView.swift [hunk 3]
```diff
@@ -117,10 +116,8 @@ class MessageContentView: BaseView {
         CATransaction.begin()
         CATransaction.setDisableActions(true)
 
-        self.bubbleView.lightGradientLayer.opacity
-        = Float(1 - focusAmount) * 0.2 * Float(1 - brightness)
-        self.bubbleView.darkGradientLayer.opacity
-        = Float(focusAmount) * 0.2 * Float(1 - brightness)
+        self.bubbleView.lightGradientLayer.opacity = 0
+        self.bubbleView.darkGradientLayer.opacity = 0.2 * Float(1 - brightness)
```
Comment (MartinJibber) [inline Shared/UI/Views/MessageContentView/MessageContentView.swift:93]: Only using the saturated purple color.

File: iOS/Flows/Conversation/Cells/ConversationMessageCellLayoutAttributes.swift [hunk 4]
```diff
@@ -15,8 +15,6 @@ class ConversationMessageCellLayoutAttributes: UICollectionViewLayoutAttributes
     var textColor: UIColor = ThemeColor.T3.color
     /// How bright the background color is. 0 is black. 1 is full brightness of the given color
     var brightness: CGFloat = 1
-    /// How in focus this cell's section is. 0 means not in focus at all. 1 means fully in focus.
-    var sectionFocusAmount: CGFloat = 1
```
Comment (MartinJibber) [inline iOS/Flows/Conversation/Cells/ConversationMessageCellLayoutAttributes.swift:19]: There's only one section and it's always in focus.

File: Shared/Flows/Onboarding/Welcome/WelcomeViewController.swift (hunk 2 above)
Comment (Benjamindodgson) [reply Shared/Flows/Onboarding/Welcome/WelcomeViewController.swift]: No we will show both. But this may no longer work... We can look at this later.

**Pull request merged**